worker/src/main/resources/application.properties

```diff
@@ -16,6 +16,7 @@ spring.integration.jdbc.initialize-schema=always
 ## --------------------------------------------------------------------------------------------------  MISC CONFIG
 file.try.lock.timeout=30
 cancellation.check.frequency=10
+report.progress.frequency=500
```

Comment:
Our smallest contract so far is 13K, which means the completion percentage can change as often as every _130_ patients processed. Let's set this property to `100`.

Reply:
done